<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingMain</class>
 <widget class="QWidget" name="SettingMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(27, 150, 208);
}
QToolButton{
Background-color:transparent;
border:noborder;
color:white;
font-size:26px;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190,210, 240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; color:#ffff00;&quot;&gt;设 置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>591</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolButton" name="parametersetting_lb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>1.参数设置</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/parameter.bmp</normaloff>:/img/parameter.bmp</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>91</width>
        <height>91</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="zerolevelsetting_lb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>2.零位设置</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/other_motor1.png</normaloff>:/img/other_motor1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>91</width>
        <height>92</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="motorsetting_lb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>3.电机设置</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/ZERO.BMP</normaloff>:/img/ZERO.BMP</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>91</width>
        <height>92</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="netsetting_lb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>4.网络设置</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/1.png</normaloff>:/img/1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>91</width>
        <height>92</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="warnsetting_lb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>5.报警屏蔽</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/Warning.png</normaloff>:/img/Warning.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>91</width>
        <height>91</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="back_lb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>6.返回</string>
      </property>
      <property name="icon">
       <iconset resource="../../Resources/picture.qrc">
        <normaloff>:/img/BACK.BMP</normaloff>:/img/BACK.BMP</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>91</width>
        <height>91</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Resources/picture.qrc"/>
 </resources>
 <connections/>
</ui>
